device/mellanox/x86_64-mlnx_msn4700-r0/plugins/sfplpmget.py

```diff
@@ -0,0 +1 @@
+../../x86_64-mlnx_msn2700-r0/plugins/sfplpmget.py
```

Comment:
need to clarify as of the 8 lanes support

Reply:
according to FW, no change for this on Spectrum-3